app/controllers/api/v3/notices_controller.rb

```diff
@@ -14,6 +14,7 @@ def create
 
     merged_params = params.merge(JSON.parse(request.raw_post) || {})
     merged_params = merged_params.merge('key' => request.headers['X-Airbrake-Token']) if request.headers['X-Airbrake-Token']
+    merged_params = merged_params.merge('key' => authorization_token) if request.headers['Authorization']
```

Comment:
How about making the condition `if authorization_token` so it doesn't replace the key if it isn't present in the Authorization header?